Add Bluetooth CGMS service UUID

wentong-li opened the pull request:
Add Bluetooth continuous glucose monitoring service (CGMS)-related uuid, including 1 service uuid and 7 characteristic uuid.
- Add BT_UUID_CGMS for Continuous Glucose Monitoring
- Add BT_UUID_CGM_MEASUREMENT for CGM Measurement
- Add BT_UUID_CGM_FEATURE for CGM Feature
- Add BT_UUID_CGM_STATUS for CGM Status
- Add BT_UUID_CGM_SESSION_START_TIME for CGM Session Start Time
- Add BT_UUID_CGM_SESSION_RUN_TIME for CGM Session Run Time
- Add BT_UUID_CGM_SPECIFIC_OPS_CONTROL_POINT for CGM Specific Ops Control Point
- Add BT_UUID_RECORD_ACCESS_CONTROL_POINT for Record Access Control Point

carlescufi commented:
@wentong-li please format your commit message as per the guidelines here:
https://docs.zephyrproject.org/latest/contribute/index.html#commit-guidelines

wentong-li commented:
The commit messages have been updated.

Thalley commented on include/bluetooth/uuid.h:
```diff
@@ -2123,6 +2123,78 @@ struct bt_uuid_128 {
+/** @def BT_UUID_CGMS_VAL
+ *  @brief Continuous Glucose Monitoring Service UUID value
+ */
+#define BT_UUID_CGMS_VAL 0x181f
+/** @def BT_UUID_CGMS
+ *  @brief Continuous Glucose Monitoring Service
+ */
+#define BT_UUID_CGMS \
+	BT_UUID_DECLARE_16(BT_UUID_CGMS_VAL)
+/** @def BT_UUID_CGM_MEASUREMENT_VAL
+ *  @brief CGM Measurement Characteristic value
+ */
+#define BT_UUID_CGM_MEASUREMENT_VAL 0x2aa7
+/** @def BT_UUID_CGM_MEASUREMENT
+ *  @brief CGM Measurement Characteristic
+ */
```
We have so far ordered our UUID entried by increasing UUID value. Please insert these in the correct places based on the values.

wentong-li replied:
Updated. They are in ascending order now.

wentong-li committed:
bluetooth: uuid: adding continuous glucose monitoring service uuids

AddBluetooth continuous glucose monitoring service (CGMS)-related uuid
BT_UUID_CGMS for Continuous Glucose Monitoring
BT_UUID_CGM_MEASUREMENT for CGM Measurement
BT_UUID_CGM_FEATURE for CGM Feature
BT_UUID_CGM_STATUS for CGM Status
BT_UUID_CGM_SESSION_START_TIME for CGM Session Start Time
BT_UUID_CGM_SESSION_RUN_TIME for CGM Session Run Time
BT_UUID_CGM_SPECIFIC_OPS_CONTROL_POINT for CGM Specific Ops Control Point
BT_UUID_RECORD_ACCESS_CONTROL_POINT for Record Access Control Point

Signed-off-by: Wentong LI <wentong.li@nordicsemi.no>

wentong-li commented:
updated the author info to meet the format according to the error message

wentong-li commented:
Updated the commit message body to meet the length limit.

wentong-li commented:
Can anyone help? Please let me know if further modification is required. Thanks!

Thalley commented:
@wentong-li Please rebase the PR :)

lairdjm commented:
@wentong-li I think something has gone a bit wrong here...

wentong-li commented:
> @wentong-li I think something has gone a bit wrong here...

Sorry for wrong operation. Just fixed...
